packages/material-ui/src/Tooltip/useKeyboardFocus.js

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+
+let prepared = false;
+let keyboardModality = false;
+
+function handleKeyDown() {
+  keyboardModality = true;
+}
+
+function handlePointerDown() {
+  keyboardModality = false;
+}
+
+export function prepare() {
+  prepared = true;
+  document.addEventListener('keydown', handleKeyDown, { passive: true });
```

Comment:
To check if it doesn't break on IE 11: https://caniuse.com/#feat=passive-event-listener.

Reply:
Tested, it's OK.

Reply:
In the  best case scenario IE11 will just cast this to a boolean and call it in the capture phase. 

TBH I'm not sure we actually need that i.e. is there a noticeable performance gain for infrequent events or is `passive` only interesting for continuous events such as scroll or mousemove?

Reply:
I agree, I don't think that using passive vs not using it matter here.